### PR TITLE
New package: VBurel.VB-CABLE version 43

### LINKED ISSUE #111373: [Package Request]: VB-CABLE
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://vb-audio.com/Cable/
Publisher: V.Burel
Package Name: VB-CABLE
Description: Virtual audio device working as virtual audio cable. All audio coming in the CABLE input is simply forwarded to the CABLE output.
Package Version: 43
Installer URL: https://download.vb-audio.com/Download_CABLE/VBCABLE_Driver_Pack43.zip

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- Resolve #111373

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115361&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

Is there a silent installer switch?

**Comment by AsciiWolf** [comment]
Good question. To be honest, I am not sure.

I have found this: https://forum.vb-audio.com/viewtopic.php?t=1764

> VBCABLE_Setup_x64.exe supports command line options like silent install with "-i -h" and uninstall with "-u -h".

File: manifests/v/VBurel/VB-CABLE/43/VBurel.VB-CABLE.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: VBCABLE_Setup.exe
+  InstallerUrl: https://download.vb-audio.com/Download_CABLE/VBCABLE_Driver_Pack43.zip
+  InstallerSha256: 66FD0A4D9F4896FF41632B7E3D53892C085C4561F53E8AE8D0F0BC10EEDD1CDD
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
```
Comment (AsciiWolf) [inline manifests/v/VBurel/VB-CABLE/43/VBurel.VB-CABLE.installer.yaml:16]:
```suggestion
    Silent: -i -h
    SilentWithProgress: -i -h

```

File: manifests/v/VBurel/VB-CABLE/43/VBurel.VB-CABLE.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: VBCABLE_Setup_x64.exe
+  InstallerUrl: https://download.vb-audio.com/Download_CABLE/VBCABLE_Driver_Pack43.zip
+  InstallerSha256: 66FD0A4D9F4896FF41632B7E3D53892C085C4561F53E8AE8D0F0BC10EEDD1CDD
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
```
Comment (AsciiWolf) [inline manifests/v/VBurel/VB-CABLE/43/VBurel.VB-CABLE.installer.yaml:26]:
```suggestion
    Silent: -i -h
    SilentWithProgress: -i -h

```

**Review by AsciiWolf — CHANGES_REQUESTED** [review]
This should fix the silent installation. Untested, but will hopefully work.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by AsciiWolf** [comment]
That's a good progress, nice! On the other hand, it probably won't be possible to auto accept this dialog since it is a security mechanism in the operating system itself.

**Comment by AsciiWolf** [comment]
@AmelBawa-msft @stephengillie Any chance this PR could get waived? As I already pointed out in my previous comment, the driver installation dialog is a dialog from Windows itself, not the installer.

**Comment by AsciiWolf** [comment]
@AmelBawa-msft @stephengillie Any chance this PR could get waived? As I already pointed out in my previous comment, the driver installation dialog is a dialog from Windows itself, not the installer.

**Comment by CoolPlayLin** [comment]
> @AmelBawa-msft @stephengillie Any chance this PR could get waived? As I already pointed out in my previous comment, the driver installation dialog is a dialog from Windows itself, not the installer.

@AsciiWolf
Unfortunately, there is a existed PR with same situation
- https://github.com/microsoft/winget-pkgs/pull/111450

**Comment by AsciiWolf** [comment]
@CoolPlayLin That's a different situation. The driver from PR you linked seems to be unsigned, while the VB-AUDIO driver is correctly signed, valid one.

**Comment by CoolPlayLin** [comment]
So is it possible to skip this driver to continue?

**Comment by AsciiWolf** [comment]
@CoolPlayLin No, it is not, but the PR that you linked is a completely different case while the VB-CABLE driver installation dialog is practically the same thing as UAC rights elevation prompt.

**Comment by CoolPlayLin** [comment]
@AsciiWolf
But machine Scope would pass the validation 🤔. This window appears it isn't UAC

**Comment by CoolPlayLin** [comment]
@AsciiWolf 
As you see, this window requires user choose whether install this driver. It appears it is launched by installer. But if this driver has installed in most of PCs, it would be waived [Validation-Unattended-Failed](https://github.com/microsoft/winget-pkgs/labels/Validation-Unattended-Failed) 😊

**Comment by AsciiWolf** [comment]
Well, it is true that the UAC elevation prompt is usually displayed before running the installer while this prompt gets displayed during the installation. So this may be the problem. Or there is an exception for the UAC prompt, but not for the driver installation prompt (that would not be ideal because it would prevent any apps containing hardware drivers from being distributed by WinGet). Let's hope someone from Microsoft will provide a more clear answer to this.

**Comment by CoolPlayLin** [comment]
@AsciiWolf 
If this driver would be independent as a silent-installation-supported software, this driver can do as a dependency for this software. The premise is that it wouldn't require user to choose whether install it again.

**Pull request closed without merging**